packages/DropdownMenu/src/components/Trigger/Trigger.js

```diff
@@ -1,38 +1,55 @@
 import React from "react";
 import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
 import Button from "@paprika/button";
 
+export const ButtonTypes = {
+  ICON: "icon",
+  RAW: "raw",
+  SIMPLE: "simple",
+};
+
+const ButtonComponentMap = {
+  icon: Button.Icon,
+  raw: RawButton,
+  simple: Button,
+};
+
+ButtonTypes.ALL = Object.values(ButtonTypes);
+
 const propTypes = {
   children: PropTypes.node,
-  icon: PropTypes.node,
-  isOpen: PropTypes.bool.isRequired,
+  buttonType: PropTypes.oneOf(ButtonTypes.ALL),
```

Comment:
changing the api of the trigger component to take a `buttonType` prop. This makes it easy for us to pass any content we like, and just change a prop to say it should show the content in a button as default or as a raw button for example which doesnt' show the button styles